src/FilterRule/NumberFormat.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Particle.
+ *
+ * @link      http://github.com/particle-php for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Particle (http://particle-php.com)
+ * @license   https://github.com/particle-php/Filter/blob/master/LICENSE New BSD License
+ */
+namespace Particle\Filter\FilterRule;
+
+use Particle\Filter\FilterRule;
+
+/**
+ * Class NumberFormat
+ *
+ * @package Particle\Filter\FilterRule
+ */
+class NumberFormat extends FilterRule
```

Comment:
Pretty much the same as before, I see how this would improve, yet I don't feel the need to. I think it's fine as it is to be honest.